scripts/integration_test_js.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
```

Comment:
this runs after the other CI test so it just has to restart the isolated server after clearing persistence